SourceX/DiabloUI/mainmenu.cpp

```diff
@@ -44,9 +44,9 @@ void mainmenu_Load(char *name, void (*fnSound)(char *file))
 	UiAddBackground(&vecMainMenuDialog);
 	UiAddLogo(&vecMainMenuDialog);
 
-	vecMainMenuDialog.push_back(new UiList(vecMenuItems, PANEL_LEFT + 64, 192, 510, 43, UIS_HUGE | UIS_GOLD | UIS_CENTER));
+	vecMainMenuDialog.push_back(new UiList(vecMenuItems, PANEL_LEFT + 64, (int) (SCREEN_HEIGHT / 2.5f), 510, 43, UIS_HUGE | UIS_GOLD | UIS_CENTER));

```

Comment:
While it's nice that you get to see more of the background, I decided to center it with the rest of the menu to better match things in-game. Also, I felt it opened up a whole can of works of how to adjust the layout to the resolution. Maybe we can look into it at a later point :)